modules/odc-accessing-developer-perspective.adoc

```diff
@@ -1,25 +0,0 @@
-// Module included in the following assemblies:
-//
-// web_console/odc-about-developer-perspective.adoc
-
-:_content-type: PROCEDURE
-[id="accessing-developer-perspective_{context}"]
-= Accessing the Developer perspective
-
-The *Developer* perspective in the {product-title} web console provides workflows specific to developer use cases.
-
-You can access the *Developer* perspective from the web console as follows:
-
-.Prerequisites
-To access the *Developer* perspective, ensure that you have logged in to the web console. The default view for the {product-title} web console is the *Developer* perspective.
-
-.Procedure
-
-. Use the perspective switcher to switch to the *Developer* perspective. The *Topology* view with a list of all the projects in your cluster is displayed.
-+
-.Developer perspective
-image::web_console_perspectives.png[The administrator and developer perspectives in the {product-title} web console.]
```

Comment:
You don't want show this image anymore?

<img width="282" alt="web_console_perspectives" src="https://user-images.githubusercontent.com/139310/210871861-b3332f95-b459-47b1-98df-3c80a9d35945.png">

Reply:
Hi @jerolimov. Yes, we decided to remove this image as other similar sections did not have such images. This was removed just to maintain consistency.